daemon/anonymous-statistics.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env sh
+
+# Valid actions:
+# - KICKSTART   - installed using the kickstart.sh script
+# - KICKSTART64 - installed using the kickstart-static64.sh script
+# - MAKESELF    - installed by running a makeself package
+# - INSTALL     - installed using netdata-installer.sh
+# - UPDATE      - updated using netdata-updater.sh
+# - FATAL       - netdata exited due to a fatal condition
+# - START       - netdata started
+
+ACTION="${1}"
+ACTION_RESULT="${2}"
+ACTION_DATA="${3}"
+
+# -------------------------------------------------------------------------------------------------
+# detect the operating system
+
+OS_DETECTION=
+NAME=
+VERSION=
+VERSION_ID=
+ID=
+ID_LIKE=
+
+if [ -f "/etc/os-release" ]; then
+    OS_DETECTION="/etc/os-release"
+    eval "$(cat /etc/os-release | grep -E "^(NAME|ID|ID_LIKE|VERSION|VERSION_ID)=")"
+elif [ -f "/etc/lsb-release" ]; then
+    OS_DETECTION="/etc/lsb-release"
+    DISTRIB_ID= DISTRIB_RELEASE= DISTRIB_CODENAME=
+    eval "$(cat /etc/lsb-release | grep -E "^(DISTRIB_ID|DISTRIB_RELEASE|DISTRIB_CODENAME)=")"
+    NAME="${DISTRIB_ID}"
+    VERSION="${DISTRIB_RELEASE}"
+    ID="${DISTRIB_CODENAME}"
+elif [ ! -z "$(which lsb_release 2>/dev/null || command -v lsb_release 2>/dev/null)" ]; then
```

Comment:
`which` is non-standard. Use `command -v` instead.

Reply:
fixed